server/repository/migrations/postgres/2022-01-27T08-00_create_changelog_table/up.sql

```diff
@@ -54,16 +54,16 @@ $$
      DECLARE
      BEGIN
         IF (TG_OP = 'DELETE') THEN
-            INSERT INTO changelog (table_name, record_id, row_action)
-              VALUES (TG_TABLE_NAME::changelog_table_name, OLD.id, 'DELETE');
+            INSERT INTO changelog (table_name, record_id, row_action, is_sync_update)
+              VALUES (TG_TABLE_NAME::changelog_table_name, OLD.id, 'DELETE', OLD.is_sync_update);
             RETURN OLD;
         ELSIF (TG_OP = 'UPDATE') THEN
-            INSERT INTO changelog (table_name, record_id, row_action)
-              VALUES (TG_TABLE_NAME::changelog_table_name, NEW.id, 'UPSERT');
+            INSERT INTO changelog (table_name, record_id, row_action, is_sync_update)
+              VALUES (TG_TABLE_NAME::changelog_table_name, NEW.id, 'UPSERT', NEW.is_sync_update);
             RETURN NEW;
         ELSIF (TG_OP = 'INSERT') THEN
-            INSERT INTO changelog (table_name, record_id, row_action)
-              VALUES (TG_TABLE_NAME::changelog_table_name, NEW.id, 'UPSERT');
+            INSERT INTO changelog (table_name, record_id, row_action, is_sync_update)
+              VALUES (TG_TABLE_NAME::changelog_table_name, NEW.id, 'UPSERT', NEW.is_sync_update);
```

Comment:
The change log is already in production. We would need a migration to update the triggers...